src/Service/InstallationService.php

```diff
@@ -1492,9 +1492,7 @@ private function createApplications(array $applicationsData): array
             }
         }//end foreach
 
-        if (empty($applicationsData) === false) {
-            $applications = $this->handleObjectType('https://docs.commongateway.nl/schemas/Application.schema.json', $applicationsData);
-        }
+        $applications = $this->handleObjectType('https://docs.commongateway.nl/schemas/Application.schema.json', $applicationsData);
```

Comment:
is there a reason to remove this if statement?

Reply:
Yup, the handleObjectType can handle empty $applicationData, but if we keep the if statement, the next line will error if no applications have been made.